docs/oceanic/index.md

```diff
@@ -50,6 +50,30 @@ NTTT - Tahiti Oceanic
 
     Honiara (AGGG) and Nauru (ANAU) FIR ATS are provided by Brisbane Oceanic (Tasman).  
 
+## Procedures
+
+### Transition Levels
```

Comment:
Would benefit from either a link, or additional table, referencing AIP ENR 1.7 2.5.2, ie QNHs below 1013 precluding cruising at certain levels. I assume the rules used in australia apply the same to other FIRs as well

Reply:
Added note